bindings/pyroot_experimental/pythonizations/src/FacadeHelpers.cxx

```diff
@@ -0,0 +1,46 @@
+// Author: Enric Tejedor CERN  04/2020
+
+/*************************************************************************
+ * Copyright (C) 1995-2020, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+// Bindings
+#include "FacadeHelpers.hxx"
+
+// Cppyy
+#include "LowLevelViews.h"
+
+// ROOT
+#include "RtypesCore.h"
+#include "ROOT/RConfig.hxx"
+
+//////////////////////////////////////////////////////////////////////////
+/// \brief Get a buffer starting at a given address
+/// \param[in] self Always null, since this is a module function.
+/// \param[in] addr Address to create buffer from
+///
```

Comment:
There's doxygen markup for the return value, I think it's `\return`?

Reply:
True! Will do that!